src/python/datastore/data_types.py

```diff
@@ -1294,6 +1294,12 @@ def _pre_put_hook(self):
                        coverage_information_key(self.fuzzer, self.date))
 
 
+class CorpusTag(Model):
```

Comment:
Do we have any design doc / one pager proposal on this? I'm just curious to brainstorm a bit more, e.g. can one fuzz target have multiple tags? The opposite is definitely possible? If we don't put multiple tags into one entity, this should probably be named more like `FuzzTargetTag` or `FuzzTargetCorpusTag`, similarly to some other entities we have (e.g. `FuzzTargetJob`). Also, we can brainstorm this more in today's or tomorrow's meeting.

Reply:
There can be multiple tags per test case. I added some more unit tests for this use case. I agree that it is a good idea to write a design doc for this! I will start it now and we can iron out more details in our meeting tomorrow.

Reply:
just to be clear, I didn't mean a real design doc, more like a one-two pager explaining the high level logic. Don't spend too much time on it, just try to write down the plan you have in mind, and others can take a look and discuss then